nss-ice-phantoms/WorkforceManagement
# Cb computers

# Description

This branch includes fixes for the details section of departments (correctly displaying the employees in the department) and sets up the views for the computers, including the details view and adding a new computer to the database.

## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout cb-computers`
1. Run BangazonWorkforce
1. Verify that the department details display properly. Also, please verify that computers list properly (including the details) and that you can make a new computer

# Checklist:
